app/Http/Controllers/V2/Dashboard/ViewTreeRestorationGoalController.php

```diff
@@ -0,0 +1,182 @@
+use App\Models\V2\Sites\SiteReport;
+use App\Models\V2\TreeSpecies\TreeSpecies;
+use App\StateMachines\EntityStatusStateMachine;
+use Carbon\Carbon;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+
+class ViewTreeRestorationGoalController extends Controller
+{
+    public function __invoke(Request $request): JsonResponse
+    {
+        $query = $this->prepareProjectQuery($request);
+        $rawProjectIds = $this->getRawProjectIds($query);
+        $allProjectIds = $this->getAllProjectIds($rawProjectIds);
+        $siteIds = $this->getSiteIds($allProjectIds);
+        $distinctDates = $this->getDistinctDates($siteIds);
+        $latestDueDate = $this->getLatestDueDate($distinctDates);
+
+        $forProfitProjectIds = $this->filterProjectIdsByType($rawProjectIds, 'for-profit-organization');
+        $nonProfitProjectIds = $this->filterProjectIdsByType($rawProjectIds, 'non-profit-organization');
+        $forProfitSiteIds = $this->getSiteIds($forProfitProjectIds);
+        $nonProfitSiteIds = $this->getSiteIds($nonProfitProjectIds);
+
+        $forProfitTreeCount = $this->treeCountByDueDate($forProfitProjectIds);
+        $nonProfitTreeCount = $this->treeCountByDueDate($nonProfitProjectIds);
+
+        $totalTreesGrownGoal = $query->sum('trees_grown_goal');
+
+        $treesUnderRestorationActualTotal = $this->treeCountPerPeriod($siteIds, $distinctDates, $totalTreesGrownGoal);
+        $treesUnderRestorationActualForProfit = $this->treeCountPerPeriod($forProfitSiteIds, $distinctDates, $totalTreesGrownGoal);
+        $treesUnderRestorationActualNonProfit = $this->treeCountPerPeriod($nonProfitSiteIds, $distinctDates, $totalTreesGrownGoal);
+
+        $averageSurvivalRateTotal = $this->getAverageSurvival($allProjectIds);
+        $averageSurvivalRateForProfit = $this->getAverageSurvival($forProfitProjectIds);
+        $averageSurvivalRateNonProfit = $this->getAverageSurvival($nonProfitProjectIds);
+
+        $result = [
+            'forProfitTreeCount' => (int) $forProfitTreeCount,
+            'nonProfitTreeCount' => (int) $nonProfitTreeCount,
+            'totalTreesGrownGoal' => (int) $totalTreesGrownGoal,
+            'treesUnderRestorationActualTotal' => $treesUnderRestorationActualTotal,
+            'treesUnderRestorationActualForProfit' => $treesUnderRestorationActualForProfit,
+            'treesUnderRestorationActualNonProfit' => $treesUnderRestorationActualNonProfit,
+            'averageSurvivalRateTotal' => floatval($averageSurvivalRateTotal),
+            'averageSurvivalRateForProfit' => floatval($averageSurvivalRateForProfit),
+            'averageSurvivalRateNonProfit' => floatval($averageSurvivalRateNonProfit),
+        ];
+
+        return new JsonResponse(ViewTreeRestorationGoalResource::make($result));
+    }
+
+    private function prepareProjectQuery(Request $request)
+    {
+
+        $query = TerrafundDashboardQueryHelper::buildQueryFromRequest($request);
+
+        return $query;
```

Comment:
Please condense into a single line - there's no need to assign to this `$query` variable before return.

Reply:
move to one single line